tests/spec/test-variables-api-service.js

```diff
@@ -67,6 +83,22 @@
                 var req = server.requests.pop();
                 req.url.should.equal('https://api.forio.com/run/forio/js-libs/;/variables/?set=a,b&include=price');
             });
+            it('should split the get in multiple GETs', function () {
+                server.requests = [];
+                var variables = ['sample_int', 'sample_string', 'sample_obj', 'sample_long', 'sample_float', 'sample_array'];
+                var include = [];
+                for (var i = 0; i < 100; i++) {
+                    include = include.concat(variables);
+                }
+
+                vs.query(include);
+                server.respond();
```

Comment:
maybe add another test to make sure the response is the right format and has all the variables?

Also another to ensure the promise fails if the first call succeeds, but the second doesn't (i.e., maybe if the last variable in the list has a typo)